packages/office-addin-dev-settings/test/unit-tests/test.ts

```diff
@@ -319,28 +319,6 @@ describe("Registration", function() {
         assert.strictEqual(first.id, firstManifestId);
         assert.strictEqual(first.manifestPath, firstRegisteredManifestPath);
       });
-      if (process.platform === "win32") {
-        it("Supports manifest path instead of id for registry value name", async function() {
-          await devSettingsWindows.registerAddIn(secondManifestPath, secondManifestPath);
-          const registeredAddins = await devSettings.getRegisterAddIns();
-          const [first, second] = registeredAddins;
-          assert.strictEqual(registeredAddins.length, 2);
-          assert.strictEqual(first.id, firstManifestId);
-          assert.strictEqual(second.id, "");
-          assert.strictEqual(first.manifestPath, firstManifestPath);
-          assert.strictEqual(second.manifestPath, secondManifestPath);
-        });
-        it("When registered by id, registry value name with manifest path is removed", async function() {
-          await devSettings.registerAddIn(secondManifestPath);
-          const registeredAddins = await devSettings.getRegisterAddIns();
-          const [first, second] = registeredAddins;
-          assert.strictEqual(registeredAddins.length, 2);
-          assert.strictEqual(first.id, firstManifestId);
-          assert.strictEqual(second.id, secondManifestId);
-          assert.strictEqual(first.manifestPath, firstManifestPath);
-          assert.strictEqual(second.manifestPath, secondManifestPath);
-        });
-      }
     });
```

Comment:
Can we add some tests for the register/publish steps with JSON files?
I know we can't automate the happy path, but we can add tests for non-existent zip files and wrong formats for example

Reply:
Hmm . . . there is some value in verifying failure cases.  What do you mean by "wrong formats"?

Reply:
By the wrong format I mean the file is not a `.zip` or it might be corrupted.
Either way, maybe the `jszip` package already handles these exemptions, but we could add tests to verify if these cases throw errors

Reply:
Since this is sending it to teams cli I'll let them check file format (we don't need to crack open the file), but I'll do some other basic checking.